src/app/plugins/controller.plugin.ts

```diff
@@ -0,0 +1,41 @@
+import { Plugin } from "../../common/plugin";
+import { IContainer, ChannelType, IMessage } from "../../common/types";
+
+export class PluginControl extends Plugin {
+  public name: string = 'controller';
+  public description: string = 'Controls activating and deactivating plugins.';
+  public usage: string = "!controller <activate | deactive> <plugin name>";
+  public permission: ChannelType = ChannelType.Admin;
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+
+  public async execute(message: IMessage, args?: string[]): Promise<void> {
+    if (!args || args!.length < 2) {
+      await message.channel.send(`Usage: ${this.usage}`);
```

Comment:
Override validate for this `validate`.
When that returns false it'll take care of sending the usage